manifests/high-availability/pdb.yaml

```diff
@@ -1,5 +1,5 @@
 ---
-apiVersion: policy/v1beta1
+apiVersion: policy/v1
```

Comment:
we shouldn't need to update that if we don't touch the supported k8s versions

Reply:
done